lib/activerecord-import/import.rb

```diff
@@ -275,7 +275,11 @@ def import_helper( *args )
           # this next line breaks sqlite.so with a segmentation fault
           # if model.new_record? || options[:on_duplicate_key_update]
             column_names.map do |name|
-              model.read_attribute_before_type_cast(name.to_s)
+              if model.class.column_defaults[name.to_s].is_a? Integer
+                model.read_attribute(name.to_s)
+              else
+                model.read_attribute_before_type_cast(name.to_s)
```

Comment:
@jkowens @zdennis  Meanwhile, I did the benchmark for `read_attribute_before_type_cast ` and `read_attribute ` which tells replacing `read_attribute_before_type_cast`  with `read_attribute` seems not be very ideal:

https://gist.github.com/aquajach/f875e13773ace44d6f85